lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -122,6 +122,12 @@ def self.filters_schema
                  has_default: true,
                  deprecated: true
 
+          schema :timestamps,
+                 type: '[]Timestamp',
```

Comment:
The type `Timestamp` is current not clearly defined in the API documentation. Adding and entry to the [section on Date, Time and Duration objects](https://www.openproject.org/docs/api/basic-objects/#dates-times-and-durations) would help.